test/complex_schemas.test.js

```diff
@@ -4,7 +4,7 @@ const convert = require('../');
 const should = require('should');
 const getSchema = require('./helpers').getSchema;
 
-['basic', 'address', 'calendar'].forEach(test => {
+['basic', 'address', 'calendar', 'single-type-arrays'].forEach(test => {
```

Comment:
These examples were taken from JSON Schema and were not meant to be the home for specific tests. 

Maybe this test file would be a better home for testing this functionality? https://github.com/wework/json-schema-to-openapi-schema/blob/master/test/invalid_types.test.js